### PR TITLE
Overrides for getopt.20120615

### PR DESCRIPTION
**Pull request by EduardoRFS**
Patch from https://github.com/fdopen/opam-repository-mingw/blob/opam2/packages/getopt/getopt.20120615/files/patch-getopt-20120615

### CODE REVIEW

**Comment by ulrikstrid** [comment]
Since the issues for Windows is in the build, I created this PR that moves to Dune: https://github.com/gildor478/ocaml-getopt/pull/1

File: packages/getopt.20120615/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	"build": [
+    [
+      "bash",
+      "-c",
+		  "#{os == 'windows' ? 'patch -p1 < patch-getopt-20120615' : 'true' }"
```
Comment (Et7f3) [inline packages/getopt.20120615/package.json]:
```suggestion
      "#{os == 'windows' ? 'patch -p1 -i patch-getopt-20120615' : 'true' }"
```

File: packages/getopt.20120615/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "build": [
+    [
+      "#{os == 'windows' ? 'patch -p1 -i patch-getopt-20120615' : 'true' }"
+    ],
```
Comment (ManasJayanth) [inline packages/getopt.20120615/package.json]:
```suggestion
    [
      "bash",
      "-c",
      "#{os == 'windows' ? 'patch -p1 -i patch-getopt-20120615' : 'true' }"
    ],
```

**Comment by Et7f3** [comment]
Why bash -c ?

**Comment by ulrikstrid** [comment]
It should probably be `sh -c`

**Pull request closed without merging**